pkg/primaza/controlplane/serviceclasses.go

```diff
@@ -35,7 +36,9 @@ func PushServiceClassToNamespaces(ctx context.Context, cli client.Client, sc pri
 		}
 
 		if err := cli.Create(ctx, sccp, &client.CreateOptions{}); err != nil {
-			return err
+			if !apierrors.IsAlreadyExists(err) {
+				return err
+			}
```

Comment:
If it already exists, don't we want to update it?

Reply:
yes, eventually this will be removed in favor of update logic. For now, it clears a bit the logs

Reply:
done